docs/guides/src/main/server/outgoinghttp.adoc

```diff
@@ -0,0 +1,147 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/kc.adoc" as kc>
+
+<@tmpl.guide
+title="Configure outgoing HTTP requests"
+summary="How to configure Keycloak's HTTP client for outgoing requests."
+includedOptions="">
+
+Keycloak often needs to make requests to the applications and services it secures. Keycloak manages these outgoing connections using an HTTP client. This guide shows how to configure the client, it's connection pool, proxy environment settings, timeouts and more.
+
+== Usage
+To configure Keycloak's outgoing Http client, use:
+
+<@kc.start parameters="--spi-connections-http-client-default-<configurationoption>=<value>"/>
```

Comment:
This can be removed as we don't have any other built-in clients. Using a custom provider should be part of the developer guide if/when we want to support that.

Reply:
This is just meant as a general usage instruction for the options under the next headline, so I don't quite get what to remove here? All the options below need `--spi-connections-http-client-default-` as prefix. This is not beautiful, but it is as it is right now. So, why remove?